telemetry/telemetry_record.py

```diff
@@ -96,9 +156,356 @@ def failed_url(url, output):
     failed_set[url] = output
     pickle.dump(failed_set, open('failed_urls', 'a'))
 
+def copy_wpr_to_benchmark():
+    """Moves wpr and .pc.wpr (all) files from local data/wpr_source to chromium
+
+    located: /src/tools/perf/page_sets/data/
+    """
+    local_path = os.path.join(os.getcwd(), 'data/wpr_source/*')
+    remote_data_path = os.path.join(CHROMIUM_SRC, 'tools/perf/page_sets/data/')
+    p = Popen('cp {0} {1}'.format(local_path, remote_data_path), shell=True)
+
+def prepare_benchmark(trial_number, num_urls):
+    """Simply checks to make sure the env is prepped before benchmarking
+
+    1. Ensures that ~/page_load_time/telemetry/temp/benchmark_results.db exists
+    2. Writes benchmarks to ~/src/tools/perf/benchmarks/page_cycler.py if they
+       do not already exist
+    NOTE: You may need to clear out the end of
+    ~/src/tools/perf/benchmarks/page_cycler.py for a clean start!
```

Comment:
Or, if we need to dynamically write out python code, we should at least write out different files other than page_cycler.py. It's a bit more work here, since you need to write out a full valid python file instead of just a snippet, but it will make things much less bug prone. We will definitely forget to clear out the end of page_cycler.py...